tests/test_backend.py

```diff
@@ -1,14 +1,29 @@
+from sys import implementation
```

Comment:
as far as I can see this `implementation` import is not used (anymore), so I would remove it

Reply:
Now removed.